src/Models/Lotto.js

```diff
@@ -0,0 +1,34 @@
+import { OutputComment } from '../Constants/display.js';
+import Errors from '../Constants/Errors.js';
+import throwError from '../Utils/throwError.js';
+import Rules from '../Utils/Rules.js';
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#validate(numbers);
+    this.#numbers = numbers.sort((a, b) => a - b);
+  }
+
+  #validate(numbers) {
+    if (Rules.isWrongLength(numbers, 6))
+      throwError(Errors.Lotto.IS_WRONG_LENGTH);
+    if (Rules.isNotNumberInNumbers(numbers))
+      throwError(Errors.Lotto.NOT_NUMBER_VALUE);
+    if (Rules.isDuplicatedValue(numbers))
+      throwError(Errors.Lotto.IS_DUPLICATE_VALUE);
+    if (Rules.isNotRangedValueInNumbers(numbers))
+      throwError(Errors.Lotto.NOT_RANGED_VALUE);
+  }
+
```

Comment:
간단한 궁금증인데요. 이 둘 중 어떤 게 읽기 편하신 것 같으신가요?
```suggestion
  #validate(numbers) {
    if (Rules.isWrongLength(numbers, 6)) throwError(Errors.Lotto.IS_WRONG_LENGTH);
    if (Rules.isNotNumberInNumbers(numbers)) throwError(Errors.Lotto.NOT_NUMBER_VALUE);
    if (Rules.isDuplicatedValue(numbers)) throwError(Errors.Lotto.IS_DUPLICATE_VALUE);
    if (Rules.isNotRangedValueInNumbers(numbers)) throwError(Errors.Lotto.NOT_RANGED_VALUE);
  }

```

Reply:
아래쪽이 더 좋아보여서, 한번 바꿔보니까 제 prettier 설정상 자동으로 줄을 바꿔버리네요 🥹

아마, 한줄에 너무 긴 길이의 코드가 있어서 관련해 설정을 좀 해봐야겠어요

Reply:
"printWidth": 100
이 항목을 보시면 좋을 것 같아요!